packages/plugin-core/src/commands/PublishExportCommand.ts

```diff
@@ -0,0 +1,85 @@
+import { NextjsExportPodUtils, PublishTarget } from "@dendronhq/pods-core";
+import _ from "lodash";
+import { DENDRON_EMOJIS } from "@dendronhq/common-all";
+import { window } from "vscode";
+import { DENDRON_COMMANDS } from "../constants";
+import { checkPreReq, NextJSPublishUtils } from "../utils/site";
+import { BasicCommand } from "./base";
+import { VSCodeUtils } from "../utils";
+
+type CommandOpts = void;
+
+type CommandOutput = {
+  nextPath: string;
+};
+
+export class PublishExportCommand extends BasicCommand<
+  CommandOpts,
+  CommandOutput
+> {
+  key = DENDRON_COMMANDS.PUBLISH_EXPORT.key;
+
+  async gatherInputs(): Promise<any> {
+    return {};
+  }
+
+  async sanityCheck() {
+    return checkPreReq();
+  }
+
+  async execute() {
+    const ctx = "PublishExportCommand";
+    this.L.info({ ctx, msg: "enter" });
+
+    const prepareOut = await NextJSPublishUtils.prepareNextJSExportPod();
```

Comment:
add a vscode.withProgress wrapper around this function

Reply:
This prompts the users for necessary inputs to move forward. Do we really want to divide their attention to the toast message when the prompt is already implying that further progression is halted until the it is submitted?

All utils in NestJSPublishUtils have withProgress wrapped around them except for ones that prompt users along the way.

Reply:
yeah, lets not do this. i missed the earlier wrapper